pkg/storage/cloud/cloud_io.go

```diff
@@ -191,6 +204,12 @@ func (r *ResumingReader) Read(p []byte) (int, error) {
 		}
 
 		if isResumableHTTPError(lastErr) {
+			retryEvent := &roachpb.ExternalStorageRetryTracingEvent{
+				Operation:      "ResumingReader.Read",
+				Retries:        int32(retries),
+				RetryableError: fmt.Sprintf("%v", lastErr),
```

Comment:
Are there any concerns about redacting the information that we record in these spans?

Reply:
hmm good point, maybe I should be using redact.Sprintf when we're saving the error string to the traces.

Reply:
I think tracing does not have redaction built-in yet - https://github.com/cockroachdb/cockroach/blob/3c946d01b713ec70a720a315cf798f6b347ab87b/pkg/util/log/trace.go#L251.

Since we are recording the error here, we can redact it ourselves.